### PR TITLE
AUX_CONNECT_RSP

### PR DESCRIPTION
**Pull request by froloffw7**

### CODE REVIEW

**Comment by sultanqasim** [comment]
Sorry, I wasn't sure when you were going to submit your PR, so I ended up implementing these two features myself last night so that I could release v1.8. Nonetheless, thank you for your contribution, including inspiring the addition of AUX_CONNECT_RSP support and a multi-line hexdump.

File: fw/RadioTask.c [hunk 1]
```diff
@@ -1279,7 +1284,10 @@ static void reactToAdvExtPDU(const BLE_Frame *frame, uint8_t advLen)
         else if (phy == PHY_2M)
             auxPeriod = (AUX_OFF_TARG_USEC + 2000) * 4;
         else // (phy == PHY_CODED_S8 || phy == PHY_CODED_S2)
-            auxPeriod = (AUX_OFF_TARG_USEC + 20000) * 4;
+            // PHY_CODED_S2:  6534us =  4542 + 150 + 1006 + 150 +  686
+            // PHY_CODED_S8: 21852us = 17040 + 150 + 2896 + 150 + 1616
+            // 22000 >= 21852
+            auxPeriod = (AUX_OFF_TARG_USEC + 22000) * 4;
```
Comment (sultanqasim) [inline fw/RadioTask.c:1290]: Good note here, I might make a patch related to this and credit you

Comment (sultanqasim) [reply fw/RadioTask.c:1290]: These timings are the longest possible advertisement, plus an AUX_CONNECT_REQ, plus an AUX_CONNECT_RSP?

**Comment by froloffw7** [comment]
Yes, this is max total duration in us, including T_IFS intervals in between..

image

**Pull request closed without merging**